packages/app/src/components/faucet/index.tsx

```diff
@@ -1,52 +1,66 @@
 import { FC } from "react";
 import { contracts } from "@tender/contracts";
 import { Button } from "grommet";
-import { useContractFunction } from "@usedapp/core";
+import { ChainId, useContractFunction, useEtherBalance, useEthers } from "@usedapp/core";
 import Dialog from "components/swap/Dialog";
 import { LoadingButtonContent } from "components/LoadingButtonContent";
 import { isPendingTransaction } from "../../utils/transactions";
-type props = {
+import stakers from "data/stakers";
+import { constants } from "ethers";
+
+const Faucet: FC<{
   symbol: string;
   protocolName: string;
-};
-
-const Faucet: FC<props> = ({ symbol, protocolName }) => {
+}> = ({ symbol, protocolName }) => {
   const { state: requestTx, send: request } = useContractFunction(contracts[protocolName].faucet, "request", {
     transactionName: `Requesting ${symbol} from faucet`,
   });
 
+  const { account } = useEthers();
+  const balance = useEtherBalance(account);
+
   // TODO why is this needed
   const requestTokens = () => {
     request();
   };
+  const isRinkebyArbitrum = stakers[protocolName].chainId === ChainId.ArbitrumRinkeby;
 
   return (
     <Dialog
       card={{ width: "large" }}
       width="large"
       openButtonLabel="Faucet"
       title="Faucet"
-      description={`Get some testnet ${symbol} and ETH (you need ETH to get ${symbol})`}
-      button1={
+      description={`Get some testnet ${symbol} and ETH (you need ETH to get ${symbol}).
+${isRinkebyArbitrum ? "To get ETH on ArbitrumRinkeby, you have to bridge it from Rinkeby." : ""}
+          `}
+      buttons={[
+        <Button
+          primary
+          style={{ width: "100%" }}
+          href="https://www.rinkebyfaucet.com/"
+          target="_blank"
+          label="Get ETH"
+        />,
+        isRinkebyArbitrum && (
+          <Button
+            primary
```

Comment:
WDYT of disabling the bridge button if you have no ETH too? 
`disabled={balance?.eq(constants.Zero)}`

Reply:
I tried that but it actually doesn't make sense - we cannot check your rinkeby eth when you are on Arbi, right?